.rubocop_todo.yml

```diff
@@ -34,3 +34,28 @@ Metrics/MethodLength:
 # Offense count: 3
 Metrics/PerceivedComplexity:
   Max: 50
+
+# TODO(qingling128) Rubocop bug.
```

Comment:
Not a bug, in fact. You were trying to use `=` to pass named parameters in Ruby, but Ruby is not Python, so that doesn't work (and treats `local_resource_id = RANDOM_LOCAL_RESOURCE_ID` as an assignment expression).
Sorry you've spent so much time on this. But none of this is actually needed. See below.

Also (for the future), the way to disable this would have been:
```ruby
        log_entry: k8s_container_log_entry(
          log_entry(0),
          # rubocop:disable Lint/UselessAssignment
          local_resource_id = RANDOM_LOCAL_RESOURCE_ID),
        # rubocop:enable Lint/UselessAssignment
```